Tools/px4airframes/srcparser.py

```diff
@@ -2,27 +2,30 @@
 import re
 import os
 
-class ParameterGroup(object):
+class AirframeGroup(object):
     """
-    Single parameter group
+    Airframe group
+
+    type: specific vehicle type (e.g. VTOL Tiltrotor, VTOL Quadrotor, etc.)
+    class: vehicle class (e.g. Multicopter, Fixed Wing, etc.)
     """
-    def __init__(self, name, af_class):
-        self.name = name
+    def __init__(self, type, af_class):
+        self.type = type
         self.af_class = af_class
-        self.params = []
+        self.airframes = []
 
 
-    def AddParameter(self, param):
+    def AddAirframe(self, airframe):
         """
         Add parameter to the group
         """
-        self.params.append(param)
+        self.airframes.append(airframe)
 
     def GetName(self):
         """
-        Get parameter group name
+        Get parameter group type
```

Comment:
Can you fix the whole comment? parameter -> airframe
And why does `GetName` return `type`?

Reply:
Sure! That's also what I wondered back then I think :laughing: 

This script in general has bad naming conventions / implicit, but I will go over them again

Reply:
Fixed!